python/paddle/fluid/framework.py

```diff
@@ -49,6 +49,12 @@
 TEMP_VAR_NAME = core.kTempVarName()
 GRAD_VAR_SUFFIX = core.kGradVarSuffix()
 ZERO_VAR_SUFFIX = core.kZeroVarSuffix()
+CONTROL_DEP_VAR_PREFIX = core.kControlDepVarName()
+
+
+def generate_control_dev_var_name():
```

Comment:
this can come out from pybind. SO we don't need to expose CONTROL_DEP_VAR_PREFIX